// @ts-ignore
async function runTask() {
    // @ts-ignore
    await Promise.resolve();
}

runTask().catch(err => console.error(err));
